src/WebJobs.Script/Description/DotNet/DotNetFunctionInvoker.cs

```diff
@@ -187,70 +185,38 @@ private void RestorePackages()
                 });
         }
 
-        public override async Task Invoke(object[] parameters)
+        protected override async Task InvokeFunction(object[] parameters, ExecutionContext context, TraceWriter traceWriter,
```

Comment:
True. That cleans things up. Let me do that.

---
In reply to: [76699592](https://github.com/Azure/azure-webjobs-sdk-script/pull/604#discussion_r76699592) [](ancestors = 76699592)